packages/@aws-cdk/aws-stepfunctions/lib/state-machine.ts

```diff
@@ -300,10 +314,12 @@ abstract class StateMachineBase extends Resource implements IStateMachine {
   /**
    * Metric for the interval, in milliseconds, between the time the execution starts and the time it closes
    *
-   * @default - sum over 5 minutes
+   * NOTE: The default this metric returns is
```

Comment:
?

Reply:
No idea what I wanted to say there. I'll take it out.